client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -47,6 +47,10 @@ const SiteName = styled.h2`
 
 const SiteUrl = styled.a`
 	color: var( --studio-gray-60 );
+	max-width: 75%;
+	text-overflow: ellipse;
```

Comment:
The property value is `ellipsis`: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow.

```suggestion
	text-overflow: ellipsis;
```

Reply:
Thanks, had it locally didn't pushed yet..